src/lib.rs

```diff
@@ -934,19 +747,16 @@ impl<T> EventListener<T> {
     ///
     /// ```
     /// use std::time::Duration;
-    /// use event_listener::Event;
+    /// use event_listener::{Event, IntoNotification, Listener};
```

Comment:
* Are these imports really needed here? I don't see a use of `Listener` or calls to `IntoListener` trait. :thinking: 
* We didn't discuss `IntoNotification` in #104 btw. I'm guessing the API we agreed on there, didn't work out exactly as planned?

Reply:
> * Are these imports really needed here? I don't see a use of `Listener` or calls to `IntoListener` trait. 🤔

Good catch, fixed

>     * We didn't discuss `IntoNotification` in [RFC: Less complex, footgun free API #104](https://github.com/smol-rs/event-listener/issues/104) btw. I'm guessing the API we agreed on there, didn't work out exactly as planned?

I'm still a fan of the current `IntoNotification` API, so I think we should keep it.

Reply:
> I'm still a fan of the current `IntoNotification` API, so I think we should keep it.

I think it's more complex than the design you proposed. With the proposed design in #104, `Listener` just becomes a trait that 2 types implement and that's it. Unless you feel very strongly about this (you're doing the work so your strong preferences trumps others :laughing:), I'd say we ask in #104 of what others think of your modified proposal.

Reply:
I'm not clear what is "modified" here? `Listener` was mentioned in #104.

Reply:
`Listener` was, yes. Not `IntoListener`. The latter implies a conversion to a listener, while the former does not and hence why I think it's a simpler concept/API.

Reply:
Having said that, if typical user doesn't have to use/import `IntoListener`, I'm not too strongly against it.

Reply:
Ah, `IntoNotification`. I think that's a separate topic entirely from this API. There are alternatives for it in the API; it's just there to allow for easily composable tagged events.